### PR TITLE
make footer work with longer content, clean up styles

### PR DESCRIPTION
**Pull request by opqdonut**
# Definition of Done / Review checklist

## Reviewability
- [x] consider adding screenshots for ease of review

### CODE REVIEW

**Comment by opqdonut** [comment]
## Normal footer
normal-footer

## Footer with lots of content
big-footer

File: src/cljs/rems/spa.cljs [hunk 1]
```diff
@@ -280,11 +280,10 @@
...
   [:footer.footer
    [:div.container
-    [:div.navbar
-     [:div.navbar-text (text :t/footer)]
-     (when (config/dev-environment?)
-       [:div.dev-only
-        [dev-reload-button]])]]])
+    (when (config/dev-environment?)
+      [:div.dev-reload-button.dev-only
+       [dev-reload-button]])
```
Comment (Macroz) [inline src/cljs/rems/spa.cljs:285]: I think it doesn't make sense to wrap `dev-reload-button` in a use site to `.dev-reload-button`, it should be instead rolled into the button component.

Comment (opqdonut) [reply src/cljs/rems/spa.cljs:285]: Do you think I should keep the `dev-only` class? It doesn't seem to have a use right now.

Comment (opqdonut) [reply src/cljs/rems/spa.cljs]: I removed it too.

**Pull request merged**